src/main/java/org/terasology/blockNetwork/BlockNetworkUtil.java

```diff
@@ -40,6 +53,13 @@ private static Rotation getRotation(Block block) {
         return rotation;
     }
 
+    /**
+    * Give the rotation of the block that has connections at defined sides
```

Comment:
`get the rotated side based off the rotation of the block.`

I think this is more clear.